antora.yml

```diff
@@ -11,8 +13,8 @@ asciidoc:
     # Fallback versions
     # We try to fetch the latest from GitHub at build time
     # --
-    full-version: 24.1.1
-    redpanda-beta-version: 24.1.7-rc1
+    full-version: 24.2.1
+    redpanda-beta-version: 24.2.1-rc5
```

Comment:
In tonight's release sync meeting, @piyushredpanda mentioned that the build would be rc7. Please doublecheck.

Reply:
I am not sure what this represents, actually. Our [beta] was v24.2.1-rc1...and all the intervening RCs until now have been ... what, release-candidates?

But yes the GA tag is going to be same as v24.2.1-rc7 tag.

Reply:
This is not ready yet. Please don't consider

Reply:
Here's the actual values 
https://github.com/redpanda-data/docs/pull/653